new feature.c

```diff
@@ -0,0 +1,5 @@
+#include<stdio.h>
+int main(){
+	printf("This is new feature");
```

Comment:
can we print `Hello World!`?

Reply:
Done

Reply:
can we remove print `This is new feature`?

Reply:
yes. done.